### ISSUE TITLE
Deprecation Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

### ISSUE THREAD

**Issue by msschl** [mdbootstrap/mdb-ui-kit#1471]
See https://sass-lang.com/documentation/breaking-changes/slash-div for more information.

```sh
LOG from ./node_modules/sass-loader/dist/cjs.js sass-loader ./node_modules/css-loader/dist/cjs.js!./node_modules/postcss-loader/dist/cjs.js!./node_modules/sass-loader/dist/cjs.js!./src/styles/index.scss
<w> Deprecation Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
<w>
<w> Recommendation: math.div($spacer, 4)
<w>
<w> More info and automated migrator: https://sass-lang.com/d/slash-div
<w>
<w> node_modules\mdb-ui-kit\src\scss\bootstrap-rtl-fix\_variables.scss 253:6  @import
<w> node_modules\mdb-ui-kit\src\scss\mdb.free.scss 8:9                        @use
<w> src\styles\_global.scss 1:1                                               @use
<w> src\styles\index.scss 1:1                                                 root stylesheet
<w>
<w> Deprecation Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
<w>
<w> Recommendation: math.div($spacer, 2)
<w>
<w> More info and automated migrator: https://sass-lang.com/d/slash-div
<w>
<w> node_modules\mdb-ui-kit\src\scss\bootstrap-rtl-fix\_variables.scss 254:6  @import
<w> node_modules\mdb-ui-kit\src\scss\mdb.free.scss 8:9                        @use
<w> src\styles\_global.scss 1:1                                               @use
<w> src\styles\index.scss 1:1                                                 root stylesheet
<w>
<w> Deprecation Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
<w>
<w> Recommendation: math.div($grid-gutter-width, 2)
<w>
<w> More info and automated migrator: https://sass-lang.com/d/slash-div
<w>
<w> node_modules\mdb-ui-kit\src\scss\bootstrap-rtl-fix\_variables.scss 349:23  @import
<w> node_modules\mdb-ui-kit\src\scss\mdb.free.scss 8:9                         @use
<w> src\styles\_global.scss 1:1                                                @use
<w> src\styles\index.scss 1:1                                                  root stylesheet
<w>
<w> Deprecation Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
<w>
<w> Recommendation: math.div($spacer, 2)
<w>
<w> More info and automated migrator: https://sass-lang.com/d/slash-div
<w>
<w> node_modules\mdb-ui-kit\src\scss\bootstrap-rtl-fix\_variables.scss 462:26  @import
<w> node_modules\mdb-ui-kit\src\scss\mdb.free.scss 8:9                         @use
<w> src\styles\_global.scss 1:1                                                @use
<w> src\styles\index.scss 1:1                                                  root stylesheet
<w>
<w> Deprecation Using / for division is deprecated and will be removed in Dart Sass 2.0.0.
<w>
<w> Recommendation: math.div($input-padding-y, 2)
<w>
<w> More info and automated migrator: https://sass-lang.com/d/slash-div
<w>
<w> node_modules\mdb-ui-kit\src\scss\bootstrap-rtl-fix\_variables.scss 715:63  @import
<w> node_modules\mdb-ui-kit\src\scss\mdb.free.scss 8:9                         @use
<w> src\styles\_global.scss 1:1                                                @use
<w> src\styles\index.scss 1:1                                                  root stylesheet
<w>
<w> 71 repetitive deprecation warnings omitted.
<w>
<w> null

webpack 5.62.2 compiled successfully in 7449 ms
```

**Comment by gbujanski** [comment]
Hi thanks for reporting. We fixed it in version 3.10.0

**Comment by ViRuSTriNiTy** [comment]
Yeah, a fix would be nice here. I'm currently bound to a specific dart sass version to avoid getting these errors.

**Comment by msschl** [comment]
For me, it is fixed in v3.10.1
